### DISCUSSION TITLE
fix the problem when setup the  blitz  and run it

### DISCUSSION THREAD

**Discussion by MohammedALREAI** [blitz-js/blitz#1445] [Q&A]
how to solve this problem 
```
\node_modules\date-fns\differenceInQuarters\index.js:1
(function (exports, require, module, __filename, __dirname) {


SyntaxError: Invalid or unexpected token]
```

**Comment by flybayer** [comment]
Is this in a monorepo? Only other place we've seen that error is in a monorepo: https://github.com/blitz-js/blitz/issues/1410

**Reply by MohammedALREAI** [reply]
what does meaning monorepo it and is there any way to solve this problem I try to setup the project from scratch  by removing all the deplandany  that  not need  so this problem  still exist i stuck  any information that can help me

**Reply by flybayer** [reply]
Monorepo is using yarn workspaces, so if you don't know what it is probably you aren't using it :)

What's the output of `blitz -vV`?

**Reply by MohammedALREAI** [reply]
yes I used yarn  to install it as global the version of the blitz is 0.24.3 so do you mean I need to reinstall as global

**Reply by flybayer** [reply]
You need to provide a lot more logs and details in order to help you. :)

By "output of `blitz -vV`", I mean copy and paste everything that gives you and put it here.

**Reply by MohammedALREAI** [reply]
this the result for blitz -vV
`debug: blitzPkgPath: F:\blitz\node_modules\blitz\dist\index.js
debug: cliPkgPath: F:\blitz\node_modules\@blitzjs\cli\lib\src\index.js

Windows 10 | win32-x64 | Node: v13.7.0

blitz: 0.25.0 (global)
blitz: 0.24.3 (local)

  Package manager: npm
  System:
    OS: Windows 10 10.0.18362
    CPU: (4) x64 AMD A8-3520M APU with Radeon(tm) HD Graphics
    Memory: 1.89 GB / 5.48 GB
  Binaries:
    Node: 13.7.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.10 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 6.14.8 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  npmPackages:
    @prisma/cli: 2.x => 2.10.2
    @prisma/client: 2.x => 2.10.2
    blitz: 0.24.3 => 0.24.3
    react: 0.0.0-experimental-7f28234f8 => 0.0.0-experimental-7f28234f8
    react-dom: 0.0.0-experimental-7f28234f8 => 0.0.0-experimental-7f28234f8
    typescript: 4.0.5 => 4.0.5`

**Comment by flybayer** [comment]
Is this error in a brand new app or did it just start happening in an app you have been working on for awhile?
